docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,72 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Calico as a Docker network plugin service
+
+When starting Calico as a Docker network plugin, the `calicoctl node --libnetwork`
+command will start up both the `calico/node` image (core Calico) and the
+`calico/node-libnetwork` image for Docker network integration.
+
+In some cases, you may want to run the `calico/node-libnetwork` Docker image as a separate service.  For example, if you choose to run `calicoctl node --detach=false --libnetwork`, Calico will start and attach to the `calico/node` container
```

Comment:
"In some cases" is a bit woolly I think.  Basically, if the user has the requirement to run Calico as a service or in upstart then this document is what you need to know.  Yadda yadda yadda.  We provide examples for systemd.